examples/ultrahdr_app.cpp

```diff
@@ -256,8 +256,8 @@ class UltraHdrAppInput {
                    uhdr_color_transfer_t hdrTf = UHDR_CT_HLG, int quality = 95,
                    uhdr_color_transfer_t oTf = UHDR_CT_HLG,
                    uhdr_img_fmt_t oFmt = UHDR_IMG_FMT_32bppRGBA1010102, bool isHdrCrFull = false,
-                   int gainmapScaleFactor = 4, int gainmapQuality = 85,
-                   bool enableMultiChannelGainMap = false, float gamma = 1.0f,
+                   int gainmapScaleFactor = 1, int gainmapQuality = 85,
```

Comment:
Thank you, I think this is great to help ensure high quality as the default (and users can of course use the flags to request lower quality where they feel it meets their size / quality objectives).